### DISCUSSION TITLE
How should banned IP's work

### DISCUSSION THREAD

**Discussion by ProfBrickz** [csmith1188/Formbar.js#297] [Q&A] [answered]
Should IPs be managed through whitelisting and blacklisting, or solely on bans?

Also should it be a text input for the ip, or a button next to the user that bans the ip, or both.

**Comment by csmith1188** [comment, marked as answer]
The Python formbar had a whitelist but it was never used. We could create a low-priority goal that adds one in later.

A ban list would be a blacklist. Ideally, we want to ban by account, but without email validation we cannot prevent new account from being made. This is where IP banning steps in to temporarily stop attacks.
